stdm/__init__.py

```diff
@@ -90,18 +90,40 @@ def main():
         help="Name of the jobset. Can choose between presubmit and continous",
     )
     parser.add_argument(
-        "--get_max_int",
+        "--get_build_number",
+        action="store_true",
+        help="Retrieve the CI build number",
+    )
+    parser.add_argument(
+        "--get_single_url",
+        action="store_true",
+        help="Retrieve a single random URL from a given build",
+    )
+    parser.add_argument(
+        "--get_all_urls",
         action="store_true",
-        help="Retrieve also the CI build number",
+        help="Retrieve all the URLs of a given build",
     )
 
     args = parser.parse_args()
 
-    print(
-        get_latest_artifact_url(
-            args.project, args.build_name, args.jobset, args.get_max_int
+    if not (args.get_all_urls ^ args.get_single_url ^ args.get_build_number):
```

Comment:
this may be confusing - it will enable `single_url` even if all the parameters are set (or none of them is). Also the following combination will enable `single_url`:
`all_urls` = True, single_url` = False, `build_number` = True
so user explicitly don't want single_url but the code will enable it

Reply:
Hmm right, I guess I'd need to make this more robust, sth like:

- If all of them are disabled, set the default to do `get_single_url`
- If two or more are enabled, return an assertion error

Reply:
Fixed